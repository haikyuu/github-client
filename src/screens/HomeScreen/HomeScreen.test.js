import React from 'react'
import renderer from 'react-test-renderer'
import { HomeScreen } from '@screens'

it('renders the HomeScreen from snapshot', () => {
	const navigation = {
		navigate: ()=>null
	}
  expect(renderer.create(
    <HomeScreen navigation={navigation}/>
  )).toMatchSnapshot()
})
