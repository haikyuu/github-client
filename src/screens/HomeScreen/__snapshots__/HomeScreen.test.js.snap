// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the HomeScreen from snapshot 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "alignSelf": "center",
        "marginTop": 15,
      }
    }
  >
    Hiring octocats ... Please wait
  </Text>
  <Image
    resizeMode="contain"
    source={1}
    style={
      Object {
        "height": 1334,
        "width": 750,
      }
    }
  />
</View>
`;
